repos/system_upgrade/common/libraries/persistentnetnames.py

```diff
@@ -42,9 +42,13 @@ def interfaces():
             attrs['name'] = dev.sys_name
             attrs['devpath'] = dev.device_path
             attrs['driver'] = dev['ID_NET_DRIVER']
-            attrs['vendor'] = dev['ID_VENDOR_ID']
-            attrs['pci_info'] = PCIAddress(**pci_info(dev['ID_PATH']))
             attrs['mac'] = dev.attributes.get('address')
+            attrs['vendor'] = dev.get('ID_VENDOR_ID', '')
+            if not attrs['vendor']:
```

Comment:
From PR description: _Non-PCI network devices (present usually on IBM Z architecture) does not have ID_VENDOR_ID attribute and also there is nothing we could put into Interface.pci_info about them._

This leads me to believe that the condition is negated.
```suggestion
            if attrs['vendor']:
```

Reply:
@MichalHe  hi \o you are right. I forgot to switch the condition when I changed the statement.